spec/factories/users.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user do
+    first_name { Faker::Name.first_name }
```

Comment:
in our app will be few stages of user 
- user with email only 
- user with email and first name && last name 

`create(:user)` will create always user with every data 

IMO it will be better that base factory user will have only email and there will be an optional trait `:with_name` or smth :)